.travis.yml

```diff
@@ -37,4 +37,4 @@ script:
   - docker-compose exec web bash -c "CI=TRUE GENERATE_REPORT=true rails test:system"
   - echo -e '<?xml version="1.0" encoding="UTF-8"?>' > output.xml
   - tail -n +2 -q ./test/reports/TEST*.xml >> output.xml
-  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then danger --verbose; fi'
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then bash -c "docker-compose -e DANGER_GITHUB_API_TOKEN=$DANGER_GITHUB_API_TOKEN exec web danger --verbose"; fi'
```

Comment:
Tried this!

Reply:
:-( didn't work...

Reply:
Here, I also added it to `docker-compose-testing.yml` but possibly other env variables are needed to?
danger --verbose doesn't report anything (but runs cleanly).